content/docs/user-guide/build/packages.md

```diff
@@ -209,3 +209,8 @@ CMake functions used in this example:
 * `ly_pip_install_local_package_editable`: See the definition in `cmake/LYPython.cmake`.
 
 {{< /note >}}
+
+## License file generation
+License attribution files can be generated during the project development process to properly attribute any 3p packages that were imported (often called the NOTICES file). To scan 3p package directories for licenses, you can run a script located in the engine's scripts\license_scanner subdirectory. This script will look for the `PackageInfo.json` file in order to create a summary file of all package licenses with SPDX tags for easy reference. 
```

Comment:
As noted previously, for consistency, use "downloadable" rather than "third-party" packages.
Also, since the tool is scanning both the project and package directories, I'd be clear about this, too.
```suggestion
License attribution files (often called the NOTICES file) can be generated during the project development process to properly attribute any code or downloadable packages that were imported. To scan project and package directories for licenses, you can run a script located in the engine's `scripts\license_scanner` subdirectory. This script will look for the `PackageInfo.json` file in order to create a summary file of all package licenses with SPDX tags for easy reference. 
```

Reply:
Seems reasonable, the project doesn't often contain a `PackageInfo.json` file (generally created for packages only)g, but it can happen.